extensions/wikia/GlobalNavigation/GlobalNavigationHelper.class.php

```diff
@@ -111,6 +111,10 @@ public function getMenuNodes() {
 			[3] // max 3 links
 		);
 
+		if ( $wgLang->getCode() !== 'en' ) {
```

Comment:
Good catch with using `self::DEFAULT_LANG`

Yeah, I was thinking exactly like you pasted. Avoiding unnecessary computations makes code faster, so let's go that way.